luigi/parameter.py

```diff
@@ -223,6 +227,23 @@ def parse(self, x):
         """
         return x  # default impl
 
+    def parse_list(self, xs):
```

Comment:
Actually. This is one of the things I like about Java vs Python. You seem to consider "private means only the class may use it". I consider it as "private means that it's Luigi-private". In particular Luigi-contributors might at any time remove it. The reason I think the way I do is that everything that's not prepended with `_` is going to end up on the [docs page](http://luigi.readthedocs.io/en/stable/api/luigi.parameter.html#luigi.parameter.Parameter).